roslibrust_test/src/ros1.rs

```diff
@@ -177,6 +151,47 @@ pub mod diagnostic_msgs {
     }
     impl ::roslibrust_codegen::RosMessageType for KeyValue {
         const ROS_TYPE_NAME: &'static str = "diagnostic_msgs/KeyValue";
+        const MD5SUM: &'static str = "cf57fdc6617a881a88c16e768132149c";
+    }
+    #[allow(non_snake_case)]
+    #[derive(
+        :: serde :: Deserialize,
+        :: serde :: Serialize,
+        :: smart_default :: SmartDefault,
+        Debug,
+        Clone,
+        PartialEq,
+    )]
+    pub struct AddDiagnosticsRequest {
```

Comment:
Request and Response types will appear to be moved because I stopped adding them to the message graph (since a type can't depend on a service request or response type) and added them to the code generation of services.